src/Umbraco.PublishedCache.NuCache/DataSource/ContentSourceDto.cs

```diff
@@ -1,47 +1,66 @@
-using System;
 using Umbraco.Cms.Core.Models;
 
-namespace Umbraco.Cms.Infrastructure.PublishedCache.DataSource
+namespace Umbraco.Cms.Infrastructure.PublishedCache.DataSource;
+
+// read-only dto
+internal class ContentSourceDto : IReadOnlyContentBase
 {
-    // read-only dto
-    internal class ContentSourceDto : IReadOnlyContentBase
-    {
-        public int Id { get; set; }
-        public Guid Key { get; set; }
-        public int ContentTypeId { get; set; }
-
-        public int Level { get; set; }
-        public string Path { get; set; } = string.Empty;
-        public int SortOrder { get; set; }
-        public int ParentId { get; set; }
-
-        public bool Published { get; set; }
-        public bool Edited { get; set; }
-
-        public DateTime CreateDate { get; set; }
-        public int CreatorId { get; set; }
-
-        // edited data
-        public int VersionId { get; set; }
-        public string? EditName { get; set; }
-        public DateTime EditVersionDate { get; set; }
-        public int EditWriterId { get; set; }
-        public int EditTemplateId { get; set; }
-        public string? EditData { get; set; }
-        public byte[]? EditDataRaw { get; set; }
-
```

Comment:
This looks like it was laid out into sections, header data, draft data, published data which makes sense but after formatting it's just mangled up props and the section comments don't make much sense anymore.